### PR TITLE
Linked configurable options for flytectl config.yaml

### PR DESCRIPTION
**Pull request by pmahindrakar-oss**
Signed-off-by: Prafulla Mahindrakar <prafulla.mahindrakar@gmail.com>

# TL;DR

Screenshot 2021-12-29 at 6 56 18 PM

Screenshot 2021-12-29 at 6 56 30 PM

## Type
- [ ] Bug Fix
- [ ] Feature
- [ ] Plugin

## Are all requirements met?

- [ ] Code completed
- [ ] Smoke tested
- [ ] Unit tests added
- [ ] Code documentation added
- [ ] Any pending items have an associated Issue

## Complete description
_How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
https://github.com/flyteorg/flyte/issues/1916

## Follow-up issue
_NA_

### CODE REVIEW

**Comment by pmahindrakar-oss** [comment]
@cosmicBboy Please have a look and see if this helps in finding all the config options to be used for config.yaml
All the configurable flags have autogenerated docs and hence they can be referenced for addition to config.yaml file

File: docs/source/index.rst [hunk 1]
```diff
@@ -48,7 +48,12 @@ on command-line. The following configuration is useful to setup.
 Basic Configuration
 --------------------
 
-Flytectl configuration
+Flytectl configuration. 
+For full list of configurable options available can be found by running ``flytectl --help`` and can be alternately be found `here <https://docs.flyte.org/projects/flytectl/en/stable/gen/flytectl.html#synopsis>`__
+
+.. NOTE::
+
+     Only Project(-p), Domain(-d), Output(-o) currently cannot be used in config file
```
Comment (cosmicBboy) [inline docs/source/index.rst:56]:
```suggestion
     Currently, the Project ``-p``, Domain ``-d``, and Output ``-o`` flags cannot be used in config file
```

**Review by cosmicBboy — DISMISSED** [review]
looks good! see suggestions on formatting the note

File: docs/source/index.rst [hunk 2]
```diff
@@ -48,7 +48,12 @@ on command-line. The following configuration is useful to setup.
 Basic Configuration
 --------------------
 
-Flytectl configuration
+Flytectl configuration.
```
Comment (cosmicBboy) [inline docs/source/index.rst:51]: doesn't seem like this line is necessary

File: docs/source/index.rst [hunk 3]
```diff
@@ -48,7 +48,12 @@ on command-line. The following configuration is useful to setup.
 Basic Configuration
 --------------------
 
-Flytectl configuration
+Flytectl configuration. 
+For full list of configurable options available can be found by running ``flytectl --help`` and can be alternately be found `here <https://docs.flyte.org/projects/flytectl/en/stable/gen/flytectl.html#synopsis>`__
```
Comment (cosmicBboy) [inline docs/source/index.rst:52]: where is this `flytectl.html` in the table of contents? I can't seem to find it from the flytectl home page: https://docs.flyte.org/projects/flytectl/en/stable/

Comment (pmahindrakar-oss) [reply docs/source/index.rst:51]: Done . thanks for pointing out. Have added to index.rst

File: docs/source/index.rst (hunk 2 above)
Comment (pmahindrakar-oss) [reply docs/source/index.rst]: yeah. I have removed this now

File: docs/source/index.rst (hunk 1 above)
Comment (pmahindrakar-oss) [reply docs/source/index.rst]: Done

**Review by cosmicBboy — APPROVED** [review]
lgtm!

**Pull request merged**